tools/measurement/apksize/apksize.gradle

```diff
@@ -25,7 +25,7 @@ apply from: "src/config/config.gradle"
 apply from: "src/database/database.gradle"
 apply from: "src/firestore/firestore.gradle"
 apply from: "src/functions/functions.gradle"
-apply from: "src/inappmessagingdisplay/inappmessaging-display.gradle"
+// apply from: "src/inappmessagingdisplay/inappmessaging-display.gradle"
```

Comment:
Is this still needed?

Reply:
It is. I commented it out during testing. Fixed.